ctp2_code/ctp/civapp.cpp

```diff
@@ -1721,6 +1721,10 @@ sint32 CivApp::InitializeGame(CivArchive *archive)
 		return FALSE;
 	}
 
+	if (archive) { // Ensure that SlicEngine's strings have correct numbering
+		SlicEngine::Reload(g_slic_filename);
+	}
```

Comment:
Doesn't that reload all there code and reset it all the internal values?

This means the internal state gets lost.

Reply:
Thanks @MartinGuehmann, this is a really good catch. It will reset the complete SlicEngine and by that will destroy what just has been loaded from the save-file. I will revert this change and create a separate issue to investigate how we can reload the strings without destroying the current state.

Reply:
Unfortunately, this need to be fixed here, as I added a new string 'Sort by age' and by that new string all save-files are broken (strings are not correctly numbered any more). I will investigate a way to load the new strings without changing the SlicEngine state.

Reply:
That's why the readme of the Apolyton edition says that you reload slic, if you want to continue your current game. In fact we have been adding new strings all the time and we did not add them add the end, but rather were it made sense for us.

So that means, this is not part of the pull request. Of course, it would be nice to have.

Reply:
Ok, in that case this PR is ready for merging as I reverted the change.

BTW: I found a solution for the renumbering of the strings. I will create a separate PR for that.